zcash_client_sqlite/src/wallet/db.rs

```diff
@@ -216,54 +223,68 @@ CREATE TABLE orchard_received_note_spends (
     UNIQUE (orchard_received_note_id, transaction_id)
```

Comment:
Lines 211-212: "It plays the same role for Orchard notes as does [`TABLE_SAPLING_RECEIVED_NOTE_SPENDS`] for Sapling notes."